protocol/proofs/fdg-refund-mode.md

```diff
@@ -0,0 +1,95 @@
+withdraw before refund mode is activated then we assume that all withdrawals could've been
+completed and there's no reason to activate refund mode for those contracts.
+
+### Enabling Blanket Refunds
+
+Existing blacklisting functionality targets specific `FaultDisputeGame` instances and cannot easily
+handle cases where many games need to be blacklisted. If we were to adopt DD#76 alongside
+[DD#77](https://github.com/ethereum-optimism/design-docs/pull/77) then we could use the proposed
+`gameCreationValidFrom` timestamp to act as an implicit blacklist of all games created before this
+timestamp. The `AnchorStateRegistry.isValidGame(...)` function would return `false` if the game is
+explicitly blacklisted OR if the game was created before this validity timestamp. All games created
+before `gameCreationValidFrom` would then enter refund mode.
+
+### FaultDisputeGame Bond Unlock Modification
+
+The `FaultDisputeGame` contract currently calls `DelayedWETH.unlock(...)` when it credits a user
+with a bond. However, `DelayedWETH.unlock(...)` is responsible for (1) starting the 7 day
+withdrawal period and (2) restricting how `FaultDisputeGame` can withdraw out of `DelayedWETH`. We
+would need to move this unlock to only be possible after the `FaultDisputeGame` has resolved.
+Additionally, this unlock would need to be re-triggered if refund mode is activated.
+
+## Risks & Uncertainties
+
+### Security
+
+`FaultDisputeGame` is, obviously, an important part of the security of the current OP Stack. Any
+modifications to the `FaultDisputeGame` contract will likely need to be audited.
+
+### AnchorStateRegistry modifications
+
+[Design Doc #76](https://github.com/ethereum-optimism/design-docs/pull/76) proposes to shift the
+state validation logic currently found within the `OptimismPortal` contract into the
+`AnchorStateRegistry`. Adoption of DD#76 would significantly simplify the specification for this
+proposal. One potential risk here is that this proposal effectively becomes a dependency of
+DD#76 as it would likely be too messy if `FaultDisputeGame` needed to reference `OptimismPortal`.
+
+### DelayedWETH Interactions
+
+We believe that the changes described here would not impact the continued ability to use
+`DelayedWETH` to act as a more serious fallback in the case that refund mode would not properly
+refund users for some reason. We should double check to confirm that we would not see any
+unexpected interactions between the two systems.
+
+### Challenger and Dispute Monitor Changes
+
+Changes that need to be made to the `FaultDisputeGame` bond unlock logic may require some small
+modifications to `op-challenger` and `op-dispute-mon`. We may be able to avoid changes to
+`op-challenger` if we move the `unlock` call into `claimCredit` and simply fail if the unlock
+has already been made. Need to confirm here that `op-challenger` will just retry this claim
+function over and over, but if that's the case then this works just fine.
```

Comment:
Challenger should just keep retrying already.  But I'm not concerned about the need to modify challenger - it has really good support for handling multiple different contract versions already so can tolerate behaviour changing.